docker-for-mac/space.md

```diff
@@ -5,90 +5,96 @@ title: Disk utilization in Docker for Mac
 
-If your client and daemon API are version 1.25 or later (use the docker version command on the client to check your client and daemon API versions.), you can display detailed space usage information with:
+3. Click **Apply & Restart** for the changes to take effect.
+
+Do not move the file directly in Finder as this can cause Docker Desktop to lose track of the file.
+
+### Delete unnecessary containers and images
+
+Check whether you have any unnecessary containers and images. If your client and daemon API are running version 1.25 or later (use the `docker version` command on the client to check your client and daemon API versions), you can see the detailed space usage information by running:
 
 ```
 docker system df -v
 ```
 
-Alternatively, you can list images with:
+Alternatively, to list images, run:
+
 ```bash
 $ docker image ls
 ```
-and then list containers with:
+
+and then, to list containers, run:
+
 ```bash
 $ docker container ls -a
 ```
 
-If there are lots of unneeded objects, try the command
+If there are lots of redundant objects, run the command:
+
 ```bash
 $ docker system prune
 ```
-This removes all stopped containers, unused networks, dangling images, and build cache.
 
-Note that it might take a few minutes before space becomes free on the host, depending
-on what format the "disk image" file is in:
-- If the file is named `Docker.raw`: space on the host should be reclaimed within a few
-  seconds.
-- If the file is named `Docker.qcow2`: space will be freed by a background process after
-  a few minutes.
+This command removes all stopped containers, unused networks, dangling images, and build cache.
+
+It might take a few minutes to reclaim space on the host depending on the format of the disk image file:
 
-Note that space is only freed when images are deleted. Space is not freed automatically
-when files are deleted inside running containers. To trigger a space reclamation at any
-point, use the command:
+- If the file is named `Docker.raw`: space on the host should be reclaimed within a few seconds.
+- If the file is named `Docker.qcow2`: space will be freed by a background process after a few minutes.
+
+Space is only freed when images are deleted. Space is not freed automatically when files are deleted inside running containers. To trigger a space reclamation at any point, run the command:
 
 ```
-$ docker run --privileged --pid=host justincormack/nsenter1 /sbin/fstrim /var/lib/docker
+$ docker run --privileged --pid=host username/nsenter1 /sbin/fstrim /var/lib/docker
 ```
 
-Note that many tools will report the maximum file size, not the actual file size.
-To query the actual size of the file on the host from a terminal, use:
+Note that many tools report the maximum file size, not the actual file size.
+To query the actual size of the file on the host from a terminal, run:
+
 ```bash
 $ cd ~/Library/Containers/com.docker.docker/Data
 $ cd vms/0   # or com.docker.driver.amd64-linux
```

Comment:
It looks like the file was moved so we need `$ cd vms/0/data`. I think we can drop the `# or com.docker.driver.amd64-linux` part.

Reply:
Fixed in commit b8594d